samples/analyze.js

```diff
@@ -355,6 +355,59 @@ function analyzeSafeSearch(gcsUri) {
   // [END analyze_safe_search]
 }
 
+function analyzeVideoTranscription(gcsUri) {
+  // [START video_speech_transcription]
+  // Imports the Google Cloud Video Intelligence library
+  const videoIntelligence = require('@google-cloud/video-intelligence')
+    .v1p1beta1;
```

Comment:
Here (and in some other places) we publish a sample that makes use of the beta endpoint, but we don't explicitly state to the user that the functionality they're using is beta. Should we be more accurate about that? There should be a clear distinction for the user between using the GA functionality and using beta functionality.

@lukesneeringer @jmdobry